cmd/beacon-chain/flags/base.go

```diff
@@ -174,6 +175,19 @@ var (
 		Usage: "The factor by which blob batch limit may increase on burst.",
 		Value: 2,
 	}
+	// DataColumnBatchLimit specifies the requested data column batch size.
+	DataColumnBatchLimit = &cli.IntFlag{
```

Comment:
There is some other places where `BlobBatchLimit` and `BlobBatchLimitBurstFactor` are used, and where the 2 new flags are not used.
With the current configuration, `batchSize` in https://github.com/0x00101010/prysm/blob/peerdas-rpc-req-resp/beacon-chain/sync/rpc_data_column_sidecars_by_root.go#L115-L116 is always 0.